### PR TITLE
test_update

### PR DESCRIPTION
**Pull request by AABur**
RE: #232 

Test for the `update` command

### CODE REVIEW

File: git_machete/tests/functional/test_update.py [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+from git_machete import cli
+from git_machete.tests.functional.commons import (GitRepositorySandbox,
+                                                  get_current_commit_hash, git,
+                                                  mock_run_cmd, popen)
+
+
+class TestMachete:
+
+    @staticmethod
+    def launch_command(*args: str) -> str:
```
Comment (PawelLipski) [inline git_machete/tests/functional/test_update.py:15]: Hmmm this should rather live in some common util class 🤔

File: git_machete/tests/functional/test_update.py [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+        with io.StringIO() as out:
+            with redirect_stdout(out):
+                with redirect_stderr(out):
+                    cli.launch(list(args))
+                    git.flush_caches()
+            return out.getvalue()
+
+    def setup_method(self) -> None:
```
Comment (PawelLipski) [inline git_machete/tests/functional/test_update.py:23]: And this as well

**Comment by AABur** [comment]
Agreed, but I want to split up a huge test_machete first. And then I'll reanalyse it.  It will be easier to do this with small test modules.

**Comment by PawelLipski** [comment]
Okay! merging then

**Pull request merged**